lib/ex_sdp/attribute/fmtp.ex

```diff
@@ -66,6 +75,14 @@ defmodule ExSDP.Attribute.FMTP do
           level_asymmetry_allowed: boolean() | nil,
           packetization_mode: non_neg_integer() | nil,
           sprop_parameter_sets: %{sps: binary(), pps: binary()} | nil,
+          # H265
+          profile_space: non_neg_integer() | nil,
+          profile_id: non_neg_integer() | nil,
+          tier_flag: non_neg_integer() | nil,
+          level_id: non_neg_integer() | nil,
```

Comment:
It seems we're missing the typespec for `interop_constraints`
```suggestion
          level_id: non_neg_integer() | nil,
          interop_constraints: non_neg_integer() | nil,
```